docs/modeling-guide.md

```diff
@@ -9,7 +9,7 @@ the basic types [here](https://www.typescriptlang.org/docs/handbook/basic-types.
 while in [behaviors](./behaviors.md) you can find the list of special interfaces used
 for describing APIs that can't be represented in the specification.
 
-Refer to the [documentation guide](doc-comments-guide.md) to add documentation to types and fields.
+Refer to the [API docs contribution guidelines](https://www.elastic.co/docs/extend/contribute/api-docs/) to add documentation to types and fields.
```

Comment:
I don't think replacing a precise link with a vague api-docs/ link is useful here. Also `doc-comments-guide.md` contains information that is missing in the API contrib guidelines.